app/models/authentication.rb

```diff
@@ -99,4 +110,18 @@ def event_prefix
     when "ExtManagementSystem" then "ems"
     end
   end
+
+  # will be removed once moving to yaml format, no need to review method
```

Comment:
What does this mean?

Are you saying these methods won't exist anymore at all?  How will the subclasses generate this format?

The main reason I'm asking is because I don't like having both `ansible`-specific and `openshift`-specific things in the `Authentication` base class.

If these methods are going away *before* we decide to merge it, and you have it here just for convenience, then I don't care.  But, if it's something you need to stick around and want to fix later, I'd rather figure out a better way of handling this now.